app/pages/user.js

```diff
@@ -16,14 +16,21 @@ class User extends Component {
     }
   }
 
+  static async getInitialProps({ query }) {
+    const { userId } = query;
+    const result = await fetch(`http://localhost:3001/${userId}/endpoints`);
```

Comment:
@IvanGuardado should we have a global variable or configuration file to avoid having this in plain text?

Reply:
For sure! I'm creating a new issue for that